src/main/relaxng/docbook/programming.rnc

```diff
@@ -1209,8 +1209,7 @@ div {
    db.union =
       element union {
          db.union.attlist,
-         db.unionname?,
-         db.programming.inlines*
+         db.unionname+
```

Comment:
Okay.

Reply:
Uhm. Surely a union is a union of types? Updated accordingly.

Reply:
An example, in case it might help (or included in a test suite), based on the first example of https://docs.julialang.org/en/v1/manual/types/#Type-Unions-1: 

```xml
<unionsynopsis language="julia">
	<unionname>IntOrString</unionname>
	
	<union>
		<type>Int</type>
		<type>String</type>
	</union>
</unionsynopsis>
```